### PR TITLE
added 6 faculty for 2018

### PR DESCRIPTION
**Pull request by ravenben**
# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [x] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -3106,7 +3107,7 @@ Donald Sheehy,University of Connecticut,http://donsheehy.net/,A5D1DF4AAAAJ
 Donatella Sciuto,Politecnico di Milano,http://home.deib.polimi.it/sciuto/,NOSCHOLARPAGE
 Donatello Conte,University of Salerno,http://videolectures.net/donatello_conte/,dAKCYJgAAAAJ
 Donatello Materassi,University of Tennessee,http://www.eecs.utk.edu/people/faculty/dmateras/,_bjIGdIAAAAJ
-Dong Jin,Illinois Institute of Technology,https://science.iit.edu/people/faculty/kevin-jin,-ln1Vp0AAAAJ
+Dong Jin,Illinois Institute of Technology,https://science.iit.edu/people/faculty/kevin-jin,#NAME?
```
Comment (emeryberger) [inline csrankings.csv:3110]: Not sure what happened here but please fix.

File: csrankings.csv [hunk 2]
```diff
@@ -6360,7 +6362,7 @@ Karrie G. Karahalios,Univ. of Illinois at Urbana-Champaign,https://cs.illinois.e
 Karrie Karahalios,Univ. of Illinois at Urbana-Champaign,https://cs.illinois.edu/directory/profile/kkarahal/,xyHrRFUAAAAJ
 Karsten Lundqvist,Victoria University of Wellington,https://ecs.victoria.ac.nz/Main/KarstenLundqvist/,aLfZJJQAAAAJ
 Karsten Weihe,TU Darmstadt,https://www.algo.informatik.tu-darmstadt.de/algorithmik/mitarbeiter/karsten-weihe/,NOSCHOLARPAGE
-Karteek Sreenivasaiah,http://www.iith.ac.in/~karteek,NOSCHOLARPAGE
+Karteek Sreenivasaiah,http://www.iith.ac.in/~karteek,NOSCHOLARPAGE,
```
Comment (emeryberger) [inline csrankings.csv:6365]: Spurious comma at the end...

**Review by emeryberger — CHANGES_REQUESTED** [review]

**Comment by emeryberger** [comment]
Ping, @ravenben

**Comment by ravenben** [comment]
sorry, international travel and putting out fires..
getting back to this
—
Ben Y. Zhao http://people.cs.uchicago.edu/~ravenben
Neubauer Professor of Computer Science, University of Chicago
PGP Key ID 0907015B

**Comment by emeryberger** [comment]
Safe to close this now?

**Comment by ravenben** [comment]
yes, please. thanks!
—
Ben Y. Zhao http://people.cs.uchicago.edu/~ravenben
Neubauer Professor of Computer Science, University of Chicago
PGP Key ID 0907015B

**Pull request closed without merging**